[2024-05-13T03:55:09.475+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-13T03:55:09.492+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v01.greet scheduled__2022-04-09T00:00:00+00:00 [queued]>
[2024-05-13T03:55:09.496+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_dag_with_python_operator_v01.greet scheduled__2022-04-09T00:00:00+00:00 [queued]>
[2024-05-13T03:55:09.497+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-13T03:55:09.502+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): greet> on 2022-04-09 00:00:00+00:00
[2024-05-13T03:55:09.505+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=3611) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-13T03:55:09.505+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_dag_with_python_operator_v01', 'greet', 'scheduled__2022-04-09T00:00:00+00:00', '--job-id', '221', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmpy6lj33s2']
[2024-05-13T03:55:09.506+0000] {standard_task_runner.py:63} INFO - Started process 3615 to run task
[2024-05-13T03:55:09.506+0000] {standard_task_runner.py:91} INFO - Job 221: Subtask greet
[2024-05-13T03:55:09.527+0000] {task_command.py:426} INFO - Running <TaskInstance: our_dag_with_python_operator_v01.greet scheduled__2022-04-09T00:00:00+00:00 [running]> on host f1ddc6e75993
[2024-05-13T03:55:09.559+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mani' AIRFLOW_CTX_DAG_ID='our_dag_with_python_operator_v01' AIRFLOW_CTX_TASK_ID='greet' AIRFLOW_CTX_EXECUTION_DATE='2022-04-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-04-09T00:00:00+00:00'
[2024-05-13T03:55:09.559+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-13T03:55:09.565+0000] {logging_mixin.py:188} INFO - Hello World
[2024-05-13T03:55:09.565+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-13T03:55:09.566+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-13T03:55:09.569+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=our_dag_with_python_operator_v01, task_id=greet, run_id=scheduled__2022-04-09T00:00:00+00:00, execution_date=20220409T000000, start_date=20240513T035509, end_date=20240513T035509
[2024-05-13T03:55:09.606+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-13T03:55:09.613+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-13T03:55:09.613+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
